bitnami/redmine/values.yaml

```diff
@@ -812,30 +757,25 @@ mailReceiver:
   ## @param mailReceiver.allowOverride Defines if email content is allowed to set attributes values. Values is a comma separated list of attributes or `all` to allow all attributes
   ##
   allowOverride: ""
+  ## @param mailReceiver.command Override default container command (useful when using custom images)
+  ##
+  command: ["bash"]
+  ## @param mailReceiver.args Override default container args (useful when using custom images)
+  ##
+  args: ["-c", "/cj/receive.sh"]
```

Comment:
If we are setting here `["bash"]` and `["-c", "/cj/receive.sh"]`,  then these values aren't really meant to override default as their description says.

I think we should move these inside the Deployment template, and leave them empty `[""]` for users who'd like to overwrite them.